test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,7 +90,8 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+        if(inputCommand.equals("add") || inputCommand.equals("delete"))
```

Comment:
Good job updating the logic test 👍 .
However, do you think the modified version helps to check the true functionality of isMutating? Since eventually all  classes having **isMutating** true would pass this test, I suggest adding another else to verify the others (e.g., checking latest updated file whether it changes or not)